src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -254,6 +254,7 @@ static void TestMain()
 Flushing
 1
 True
+True
```

Comment:
You are correct, transformation of code coverage instrumentation to this architecture is still in the future. The instrumentation point is introduced for implicit constructor initializer of an explicitly declared constructor, we create a sequence point for it. If you don't think it should be instrumented for code coverage, I can suppress that. Let me know.